_data/people_currentcrew_generalmembers.json

```diff
@@ -6,11 +6,7 @@
     {
         "first_name": "Alice",
         "last_name": "Ho"
-    },
-    {
-        "first_name": "Mason",
-        "last_name": "Xiao"
-    },
+    }
```

Comment:
You may need a comma here!

Reply:
Uhh I just realised my commit overrode all the changes I did before so this pull request should probably be deleted

Reply:
There isn't an undo button anywhere is there?

Reply:
Feel free to close it or push more commits!

Reply:
Ok I realised what I did, closing pull request